scripts/bash/run_enceladus.sh

```diff
@@ -272,6 +272,14 @@ echoerr() {
     echo "$@" 1>&2;
 }
 
+get_temp_log_file() {
+    DATE=`date +%Y_%m_%d-%H_%M_%S`
+    NAME=`sed -e 's#.*\.\(\)#\1#' <<< $CLASS`
```

Comment:
Everything works, and seems fine. I am just curious. We want everything after the last dot, right?
can I ask why the `\(\)`?

Reply:
It is used to specify which subpattern to keep.
But yeah, after a careful look this can can be simplified:
```
NAME=`sed -e 's#.*\.##' <<< $CLASS`
```

Reply:
If you are planning to change it, go for it. If not just merge you already have two approvalz